cmd/subnetcmd/addValidator.go

```diff
@@ -159,7 +182,22 @@ func addValidator(cmd *cobra.Command, args []string) error {
 		return err
 	}
 	deployer := subnet.NewPublicDeployer(app, useLedger, kc, network)
-	return deployer.AddValidator(subnetID, nodeID, weight, start, duration)
```

Comment:
Shouldn't we be checking if threshold > 1? If it is == 1, we should keep the old behavior.